plugins/dopf/core/dsqlbuilder.pas

```diff
@@ -255,26 +260,32 @@ class function TdGUpdateBuilder.MakeFields(ATable: T; out AFields,
     if X > -1 then
     begin
       P := ATable.PrimaryKeys[X];
-      AParams += P + ' = :' + P + ' and ';
+      if AFieldQuote=#0 then
```

Comment:
Yes, so the code looks more concise. But ofcourse in case `string` `FieldQuote`

Do you want to use the `string` type for this property for this reason ?

Reply:
Code refactoring is always welcome. :-)